projects/npmjs.com/npx-shim

```diff
@@ -1,10 +1,12 @@
 #!/bin/sh
 
+set -x
+
 # or npx fails ※
 # FIXME a patch to npm here then upstream would be preferable
 mkdir -p ~/.local/lib
 
-d="$(cd "$(dirname $0)" && pwd)"
-"$d"/npm exec "$@"
+d="$(cd "$(dirname $0)"/../tbin && pwd)"
+"$d"/npx "$@"
```

Comment:
npx is just an alias for `npm exec`. It seems like calling that directly is safer. Dunno.

Reply:
true, but it looks like it sets some env. just trying to see what might be the cause of https://github.com/teaxyz/pantry/actions/runs/5337582095